sklearn/preprocessing/tests/test_function_transformer.py

```diff
@@ -115,4 +115,24 @@ def test_kw_arg_reset():
 
     # Test that rounding is correct
     testing.assert_array_equal(F.transform(X),
-                                  np.around(X, decimals=1))
+                               np.around(X, decimals=1))
+
+
+def test_inverse_transform():
+    X = np.array([1, 4, 9, 16]).reshape((2, 2))
+
+    # Test that inverse_transform works correctly
+    F = FunctionTransformer(func=np.sqrt, inverse_func=lambda x: x**2)
```

Comment:
This test is _okay_ but we would be more explicitly testing the functionality if we had an `inverse_func` that was not a true inverse. This test could be passed by a `FunctionTransformer` that memorised its input. It's pretty minor, but I wouldn't generally test with the most usual case when unusual cases are to be supported.